packages/apollo-link-schema/src/schemaLink.ts

```diff
@@ -12,25 +12,18 @@ export namespace SchemaLink {
      * The root value to use when generating responses.
      */
     rootValue?: any;
-
-    /**
-     * A context to provide to resolvers declared within the schema.
-     */
-    context?: any;
   }
 }
 
 export class SchemaLink extends ApolloLink {
   public schema: GraphQLSchema;
   public rootValue: any;
-  public context: any;
 
-  constructor({ schema, rootValue, context }: SchemaLink.Options) {
```

Comment:
I realized that this wasn't documented in the first place.

Reply:
It wasn't documented but it was provided. This is technically a breaking change. Could it support both cases initially? If not, I think it should be a major bump.

Reply:
On second thought, I think supporting both cases would be more confusing than it's worth. I'm 👍 for this change, but I still think it should be a major bump.